docs/tutorial.md

```diff
@@ -18,6 +18,83 @@ The wizard will also ask if you'd like it setup any additional metadata files su
 > Ref: See [`zigmod init`](./commands/init.md) for more info.
 
 ---
+## Understanding `zig.mod`
+
+Running `zigmod init` will ask if You want to create an application or a
+library. Based on You answer the resulting `zig.mod` file have two types.
+
+### `zig.mod` for an application
+
+```yaml
+id: ucst208pgxu7c1slqop2ng9llcjy4369zmupp21wxjvv0nib
+name: my-package
+license: None
+description: None
+root_dependencies:
+  - src: git https://github.com/jecolon/ziglyph
+```
+
+In this case You can use `@import("ziglyph")` to import the package and use it.
+
+### `zig.mod` for a library
+
+```yaml
+id: ucst208pgxu7c1slqop2ng9llcjy4369zmupp21wxjvv0nib
+name: my-package
+main: src/lib.zig
+license: None
+description: None
+dependencies:
+  - src: git https://github.com/jecolon/ziglyph
+```
+
+Here I assume that You initialized Your project with `zig init-lib`, so Your
+`build.zig` the `addStaticLibrary()` call points to `src/main.zig`.
```

Comment:
perhaps it may be important to point out that `zig init-lib` and `zigmod init` with `lib` have no association. that's likely a bad name job on my part, but zig's lib refers to static and dynamic archive/library files (eg `.so`, `.a`) and zigmod's lib refers to a package importable by other packages and applications